mesh/pncmesh.cpp

```diff
@@ -1031,15 +1044,13 @@ void ParNCMesh::GetFaceNeighbors(ParMesh &pmesh)
    pmesh.send_face_nbr_elements.MakeFromList(nranks, send_elems);
 
    // go over the shared faces again and modify their Mesh::FaceInfo
-   for (int i = 0; i < shared.conforming.Size(); i++)
+   for (const auto& cf : shared.conforming)
    {
-      const MeshId &cf = shared.conforming[i];
       Face* face = GetFace(elements[cf.element], cf.local);
-
       Element* e[2] = { &elements[face->elem[0]], &elements[face->elem[1]] };
       if (e[0]->rank == MyRank) { std::swap(e[0], e[1]); }
 
-      Mesh::FaceInfo &fi = pmesh.faces_info[cf.index];
+      auto &fi = pmesh.faces_info[cf.index];
```

Comment:
But this again makes the reader do more work if they need to figure out the type.

Reply:
fixed in https://github.com/mfem/mfem/pull/3587/commits/1e8ede837ac5f02a7c8dde34b2469311c8e13992